openfe/setup/methods/openmm/equil_rbfe_methods.py

```diff
@@ -437,7 +438,69 @@ def dG_error(self):
         return error
 
 
-class RelativeLigandTransform(FEMethod):
+class RelativeLigandTransform(gufe.Protocol):
```

Comment:
This new class is the split we introduced between the Protocol (i.e. description of experiment) and the Transformation (the experimental method applied/parametrised with a specific chemical system)